firebase-common/api.txt

```diff
@@ -91,3 +106,10 @@ package com.google.firebase.provider {
 
 }
 
+package com.google.firebase.util {
+
+  public final class RandomUtilKt {
```

Comment:
Could you please share with me what's the use of this class?

Reply:
I added that class last week: https://github.com/firebase/firebase-android-sdk/pull/5818

This class should not be part of the public API though. It's the Java class that Kotlin generates for the free functions defined in `RandomUtil.kt`.

@daymxn Do you know how to exclude this?

Reply:
Bit of a weird kotlin-java interop issue. You need to have a package level hide for `com.google.firebase.util`. 

Eg; having a `package-info.java` file in the `util` folder with:
```java
/** @hide */
package com.google.firebase.util;
```

Reply:
Hmm, would you like to do this as part of this PR?

Reply:
Consider it done:)